AmplifyPlugins/Storage/Tests/AWSS3StoragePluginTests/Operation/AWSS3StorageUploadFileOperationTests2.swift

```diff
@@ -0,0 +1,81 @@
+////
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+@testable import Amplify
+@testable import AmplifyTestCommon
+@testable import AWSPluginsTestCommon
+@testable import AWSS3StoragePlugin
+@testable import AWSPluginsCore
+
+import AWSS3
+import XCTest
+
+final class AWSS3StorageUploadFileOperationTests2: AWSS3StorageOperationTestBase {
```

Comment:
Similar to what happened to me in v1, adding this test to `AWSS3StorageUploadFileOperationTests` made the linter complain about file length. In addition, naming this file anything more meaningful resulted in the linter complaining about the class name length. I'm happy to rename and add the linter suppression if folks prefer that.